compiler/hash-codegen/src/lower/intrinsics.rs

```diff
@@ -114,7 +114,7 @@ impl<'a, 'b, Builder: BlockBuilderMethods<'a, 'b>> FnBuilder<'a, 'b, Builder> {
     ) {
         let src = self.codegen_operand(builder, src);
 
-        self.ctx.layouts().map_many_fast([src.info.layout, dest.info.layout], |layouts| {
+        self.ctx.layouts().store().map_many_fast([src.info.layout, dest.info.layout], |layouts| {
```

Comment:
Is it needed to still have `.store()` on layouts? You can just use `layout_stores()` right?